test/system/alpha/tooltip_test.rb

```diff
@@ -66,6 +66,42 @@ def test_appends_to_existing_aria_describedby
       assert_equal("existing-description-id #{tooltip_id}", find("button")["aria-describedby"])
     end
 
+    def test_does_not_render_duplicate_describedby_id_when_attribute_callback_triggered
+      visit_preview(:description_tooltip_on_button_with_existing_describedby)
+
+      tooltip_id = find("tool-tip", visible: :hidden)["id"]
+      assert_equal("existing-description-id #{tooltip_id}", find("button")["aria-describedby"])
+      evaluate_script("(function(el) {
+        return (
+          el.setAttribute('data-type', 'label')
+        );
+      })(arguments[0]);", find("tool-tip", visible: :hidden))
+      evaluate_script("(function(el) {
+        return (
+          el.setAttribute('data-type', 'description')
+        );
+      })(arguments[0]);", find("tool-tip", visible: :hidden))
+      assert_equal("existing-description-id #{tooltip_id}", find("button")["aria-describedby"])
+    end
+
+    def test_does_not_render_duplicate_labelledby_id_when_attribute_callback_triggered
+      visit_preview(:label_tooltip_on_button_with_existing_labelledby)
+
+      tooltip_id = find("tool-tip", visible: :hidden)["id"]
+      assert_equal("existing-label-id #{tooltip_id}", find("button")["aria-labelledby"])
+      evaluate_script("(function(el) {
+        return (
+          el.setAttribute('data-type', 'description')
+        );
+      })(arguments[0]);", find("tool-tip", visible: :hidden))
+      evaluate_script("(function(el) {
+        return (
+          el.setAttribute('data-type', 'label')
+        );
+      })(arguments[0]);", find("tool-tip", visible: :hidden))
+      assert_equal("existing-label-id #{tooltip_id}", find("button")["aria-labelledby"])
+    end
+
```

Comment:
These same tests will fail in `main` with an ID output like:

`"existing-description-id tooltip-1663619871156-8889 tooltip-1663619871156-8889"`